scripts/extract-package-code.R

```diff
@@ -126,3 +138,101 @@ extract_package_vignettes <- function(pkg, pkg_dir, output_dir) {
 
     vignettes
 }
+
+
+path_sanitized <- function(path) {
+    path_abs(path_tidy(path), getwd())
+}
+
+parse_command_line_arguments <- function() {
+
+    option_list <- list(
+        make_option(c("-r", "--r-dyntrace"), action="store", type="character",
```

Comment:
I used to work with Don Hopkins and he always said - put a lot of white space in your code, to help the eyes to rest a bit...

How about this formatting of the long argument lists?
```r
make_option(
  c("-r", "--r-dyntrace"), 
  action="store",
  type="character",
  default="../../R-dyntrace/bin/R", help="R-dyntrace interpreter.",
  metavar="r-dyntrace"
),
```

Reply:
And rest your eyes shall get!

Reply:
Thanks!